apps/ios/Shared/Views/ChatList/ServersSummaryView.swift

```diff
@@ -0,0 +1,571 @@
+                        primaryButton: .destructive(Text("Reset")) {
+                            Task {
+                                do {
+                                    try await resetAgentServersStats()
+                                    getServersSummary()
+                                } catch let error {
+                                    alert = SomeAlert(
+                                        alert: mkAlert(
+                                            title: "Error resetting statistics",
+                                            message: "\(responseError(error))"
+                                        ),
+                                        id: "error resetting statistics"
+                                    )
+                                }
+                            }
+                        },
+                        secondaryButton: .cancel()
+                    ),
+                    id: "reset statistics question"
+                )
+            } label: {
+                Text("Reset statistics")
+            }
+        }
+    }
+
+    private func getServersSummary() {
+        do {
+            serversSummary = try getAgentServersSummary()
+        } catch let error {
+            logger.error("getAgentServersSummary error: \(responseError(error))")
+        }
+    }
+}
+
+struct SubscriptionStatusView: View {
+    @EnvironmentObject var m: ChatModel
+    var activeSubs: Int
+    var pendingSubs: Int
+
+    var body: some View {
+        let netInfo = m.networkInfo
+        if netInfo.online {
+            let (image, color, variableValue, opacity) = networkOnlineImage(netInfo.networkType)
+            if #available(iOS 16.0, *) {
+                Image(systemName: image, variableValue: variableValue)
+                    .foregroundColor(color)
+            } else {
+                Image(systemName: image)
+                    .foregroundColor(color.opacity(opacity))
+            }
+        } else {
+            Image(systemName: "wifi.slash")
+                .foregroundColor(.secondary)
+        }
+    }
+
+    func networkOnlineImage(_ networkType: UserNetworkType) -> (String, Color, Double, Double) {
+        switch networkType {
+        case .cellular:
+            let (color, variableValue, opacity) = cellularbarsColor
+            return ("cellularbars", color, variableValue, opacity)
+        default:
+            let (color, variableValue, opacity) = wifiColor
+            return ("wifi", color, variableValue, opacity)
+        }
+    }
+
+    var wifiColor: (Color, Double, Double) {
+        if activeSubs > 0 {
+            let wifiVariableValue = ( // wifi has 3 sections
+                activeSubsPercentage >= 1 ? 1
+                : (activeSubsPercentage >= 0.5 && activeSubsPercentage < 1) ? 0.6
+                : (activeSubsPercentage > 0 && activeSubsPercentage < 0.5) ? 0.3
+                : 0
```

Comment:
in case of ChatListView indicator, maybe we should apply activeSubsPercentage directly, in that case "wifi" would be fully colored starting with 68% and "cellularbars" - with 76%